pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -213,9 +213,9 @@ func (dsw *desiredStateOfWorld) AddPodToVolume(
 		volumeName = volumehelper.GetUniqueVolumeNameForNonAttachableVolume(podName, volumePlugin, volumeSpec)
 	}
 
-	volumeObj, volumeExists := dsw.volumesToMount[volumeName]
+	_, volumeExists := dsw.volumesToMount[volumeName]
 	if !volumeExists {
-		volumeObj = volumeToMount{
+		volumeObj := volumeToMount{
```

Comment:
can you just directly assign this to dsw.volumesToMount[volumeName] rather than creating a new variable?